docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -364,40 +439,174 @@ Dagster Cloud currently includes support for four levels of role-based access co
+
+- The user is not **Organization Admin**, and
+- The current user role for the deployment is **Viewer** or **Launcher**
+
+For example, if a user is assigned the **Viewer** user role for a deployment, only a more permissive role can be assigned for a code location in the deployment. Specifically, an **Editor** or **Launcher** user role:
+
+<Image
+alt="Overriding the Viewer user role for a code location"
+src="/images/dagster-cloud/user-token-management/code-location-override.png"
+width={538}
+height={368}
+/>
+
+If a user is an **Organization Admin** or a deployment-level **Admin** or **Editor**, they already have the most permissive user role possible for the code location. In this case, an override is unnecessary and the **Edit user role** button next to the code location will be greyed out:
+
+<Image
+alt="Override for a code location not allowed"
+src="/images/dagster-cloud/user-token-management/not-allowed-code-location-override.png"
+width={1035}
+height={466}
+/>
+
+To remove an override, click **Edit user role** next to the code location to open the user role selection dialog. Then, click the **Remove override** button:
+
+<Image
+alt="Removing the override for a code location"
+src="/images/dagster-cloud/user-token-management/remove-code-location-override.png"
+width={536}
+height={383}
+/>
+
+#### Branch deployment permissions
+
+Granting access to Branch Deployments gives users access to all Branch Deployments for all the code locations they have access to. Consider the following example:
+
+<Image
+alt="User roles for prod, staging, and all Branch Deployments"
+src="/images/dagster-cloud/user-token-management/branch-deployment-example.png"
+width={898}
+height={400}
+/>
+
+In this case, the user would have **Editor** access to Branch Deployments for all code locations in the `prod` deployment, but not in `staging`. This is because they're a **Viewer** for the code locations in the `prod` deployment, but they don't have access to the `staging` deployment or any of its code locations.
+
+---
+
+## Adding users
+
+<Note>
+  <strong>Organization Admin</strong> or <strong>Admin</strong> permissions are
+  required to add users in Dagster Cloud.
+</Note>
+
+Before you start, note that:
+
+- **If using Google for SSO**, users must be added in Dagster Cloud before they can log in.
+
+- **If using a SAML-based solution like Okta**, users must be assigned to the Dagster app in the SSO portal to log in. By default, users will be granted Viewer permissions on each deployment. The default role can be adjusted by modifying the [`sso_default_role` deployment setting](/dagster-cloud/developing-testing/deployment-settings-reference#sso-default-role).
+
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Cloud Settings**.
+3. Click the **Users** tab.
+4. In the **Enter an email...** field, enter the user's email address.
+5. Click **Add user**. The user will be added to the list of users.
```

Comment:
the starting permissions are the lowest permission that you're allowed to set, in any deployment that you're allowed to set it for (so Standard adds as Editor, Enterprise adds as Viewer) - not sure we need to say that here though

Reply:
I think it's okay to leave it out for now - we can come back and add it in if there's any confusion or feedback about it